velox/exec/fuzzer/AggregationFuzzerBase.cpp

```diff
@@ -319,7 +330,16 @@ std::vector<RowVectorPtr> AggregationFuzzerBase::generateInputDataWithRowNumber(
     }
 
     for (auto i = children.size(); i < types.size() - 1; ++i) {
-      children.push_back(vectorFuzzer_.fuzz(types[i], size));
+      if (partitionKeySet.find(names[i]) != partitionKeySet.end()) {
+        // The partition keys are built with a dictionary over a smaller set of
+        // values. This is done to introduce some repetition of key values for
+        // windowing.
+        auto baseVector = vectorFuzzer_.fuzz(types[i], numPartitions);
+        children.push_back(
+            BaseVector::wrapInDictionary(nullptr, indices, size, baseVector));
```

Comment:
Why is there a  nullptr for the nulls buffer ? We could add null values for the partition as well

Reply:
Hi @aditi-pandit and @pramodsatya, Do you mean the default value of nullRatio in VectorFuzzer::Options? I think we just set it to 0 by default. But when WindowFuzzerTest is run, this nullRatio option is overwritten by the --null_ratio command-line argument which is 0.1 by default.

Reply:
Got it, thanks for the clarification @kagamiori.